src/hooks/useForceSync.js

```diff
@@ -0,0 +1,35 @@
+import { useCallback } from 'react'
+
+function useForceSync({ doc, serviceClient, collabProvider }) {
+  const noteId = doc && doc.file.id
+
+  // Be sure to save everything before leaving
+  // and force sync with the io.cozy.file
+  const forceSync = useCallback(
+    async function forceSync() {
```

Comment:
Nit but out codebases usually have anonymous functions in this sort of cases.

Reply:
Yep, that's a few times that I wanted to raise the subject.

A plain old function allows us to have a named function in our debug tools and call stack, with no additionnal shortcomming that I know of.

It also remove the need to add an explicit displayName for React components.

Did I miss something that should make us use anonymous functions ?

Reply:
> It also remove the need to add an explicit displayName for React components.

Only when they are wrapped in an HOC at assignment time, otherwise the displayName is correctly inferred from the variable name.

Apart from this, I have no problem personally with the use of named function expressions.

Reply:
TIL, thank you